content/docs/learning-guide/tutorials/extend-the-editor/shape-example-py.md

```diff
@@ -0,0 +1,508 @@
+---
+linkTitle: Create a Custom Tool Gem in Python
+title: Create a Custom Tool Gem in Python to Extend Open 3D Engine Editor
+description: Learn how to extend the Open 3D Engine (O3DE) Editor by creating a custom tool Gem in Python.
+weight: 300
+toc: true
+---
+
+In this tutorial, you create a custom tool Gem written in Python, called **Shape Example**, that extends the **Open 3D Engine (O3DE) Editor**. The Shape Example tool allows you to create entities with a Shape component and configure their component properties. You'll learn how to use the `PythonToolGem` template and practice Python development with [Qt](https://wiki.qt.io/Main), O3DE Tools UI API, and other O3DE APIs.
```

Comment:
Many of the same comments in the cpp tutorial apply to the Python tutorial, too.